crates/librqbit_core/src/magnet.rs

```diff
@@ -62,6 +88,10 @@ impl Magnet {
                 id20,
                 id32,
                 trackers,
+                select_only: match files.is_empty() {
```

Comment:
nit: this probably would be a tiny bit less verbose with just and if statement. Please replace with an if

Reply:
such as
```
                select_only: if files.is_empty() {
                    None
                } else {
                    Some(files)
                },
```?

I think I disagree, and prefer
```
                select_only: match files.is_empty() {
                    true => None,
                    false => Some(files),
                },
```

But if that is blocking for you, I can apply it. LMK

Reply:
To avoid bikeshedding, I'll give you a better argument: code style consistency.

The matter itself isn't important, and there's no "right" solution. However consistent code style is preferable. Because I prefer the "if" version, and I wrote most of rqbit, I would believe that "if" style is used throughout.

So then at least for the sake of consistency please change to "if"

Reply:
applied :) didn't have a strong preference, but my default is match over if :D https://github.com/ikatson/rqbit/pull/248/commits/db420f3a521c77a4003bb423d3cd7a67a25253e4